test/StreamManagerTest.js

```diff
@@ -0,0 +1,769 @@
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+const { time, loadFixture } = require("@nomicfoundation/hardhat-network-helpers");
+const { getSignersAndDeployContracts } = require("./fixtures");
+
+describe.only("StreamManager:", async () => {
+	const amount = 1000
+    const rate = 1500
+    const terminationPeriod = 18 * 24 * 3600; // 18 days
+    const cliffPeriod = 24 * 3600; // 24 hrs
+    const zero = ethers.constants.AddressZero 
+
+	it("Checking fixtures", async () => {
+		await loadFixture(getSignersAndDeployContracts)
+
+	    // Returning the `decimals` for coverage*
+	    const { mockUSDT } = await loadFixture(getSignersAndDeployContracts)
+	    expect(await mockUSDT.decimals()).to.eq(6)
+  	})
+	
+	describe("changeCommissionAddress();", async () => {
+		// Tests for `changeCommissionAddress();`
+		// Changing the address of the commission
+		it('Chainge address fee: address of the admin is changing', async () => {
+	    	
+	    	const { admin, payee1, streamManager } = await loadFixture(getSignersAndDeployContracts)
```

Comment:
@CTAPCKPIM I like your style of writing tests.

Try to write tests starting from:

How the test cases are declared in the function

1. First modifiers
2. Then one by one the revert statement
3. Success case
4. Case for second trigger (if it's available)